internal/bminventory/inventory.go

```diff
@@ -478,7 +478,7 @@ func (b *bareMetalInventory) GenerateClusterISO(ctx context.Context, params inst
 		cluster.ProxyHash == clusterProxyHash {
 		var err error
 		imgName := getImageName(params.ClusterID)
-		imageExists, err = b.s3Client.UpdateObjectTag(ctx, imgName, "create_sec_since_epoch", strconv.FormatInt(now.Unix(), 10))
+		imageExists, err = b.s3Client.UpdateObjectTimestamp(ctx, imgName)
```

Comment:
Maybe this all interface should be object handler interface? 
i mean without the s3 signature.

Reply:
Changed to objectHandler